### DISCUSSION TITLE
Bypass collectionNameRegex

### DISCUSSION THREAD

**Discussion by theMackabu** [pocketbase/pocketbase#1649] [Q&A] [answered]
Is it possible to bypass `collectionNameRegex` when manually creating a collection with `forms.NewCollectionUpsert`? I would like to use other characters, such as dots and dashes, which is [supported](https://stackoverflow.com/questions/3694276/what-are-valid-table-names-in-sqlite) when putting quotes around the name. The current regex does not allow that to happen

**Comment by ganigeorgiev** [comment, marked as answer]
No, this is not supported. `forms.CollectionUpsert` applies validations. If you don't want validations then you can use directly the `Dao` - https://pocketbase.io/docs/collection-methods/#create-new-collection-without-data-validations.

But in general, I don't recommend using a different characters for the collection names outside of the allowed ones, especially the dot character, because it may not work with the nested filters and expand parsing.

**Reply by theMackabu** [reply]
Alright, I might create a field with the name instead
